helm/conjur-app-deploy/charts/app-secrets-provider-rotation/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for the Application with Secrets Provider init container in Push-to-File mode Helm chart
+# This is a YAML-formatted file.
+
+global:
+  conjur:
+    # name of the ConfigMap created by the conjur-config-namespace-prep chart
+    conjurConnConfigMap: "conjur-connect"
+  appServiceType: "NodePort"
+
+app:
+  image:
+    repository: "docker.io/cyberark/demo-app"
+    tag: "latest"
+    # supported values: "Always", "IfNotPresent", "Never"
+    pullPolicy: "Always"
+  platform: "kubernetes"
+  secretsMountPath: "/mounted/secrets"
+
+secretsProvider:
+  image:
+    repository: "docker.io/cyberark/secrets-provider-for-k8s"
```

Comment:
I think it might be a good idea to change this to your `szheigh/secrets-provider-for-k8s` for now.
At the very least, we might want to put a comment here indicating that the run won't work with the official `cyberark` image.

Reply:
In truth, this test runs against the 'edge' docker tag, so it would work if the CI was running successfully on cyberark/secrets-provider-for-k8s#main. Alas right now it's not...

Reply:
@john-odonnell secrets-provider's CI is now working so this is no longer an issue. Can you please verify that `./bin/test-workflow/start -a secrets-provider-rotation` succeeds locally for you?

Reply:
Success! Just needs a rebase and I'll approve.

Reply:
@john-odonnell Rebase complete